Makefile

```diff
@@ -289,13 +289,21 @@ vendor:
 	$(GO) mod vendor
 	$(GO) mod verify
 
-.PHONY: vendor-in-container
-vendor-in-container:
-	podman run --rm --env HOME=/root \
+
+# We define *-in-container targets for the following make targets. This allow the targets to be run in a container.
+# Note that the PODMANCMD can also be overridden to allow a different container CLI to be used on systems where podman is not already available.
+IN_CONTAINER_TARGETS = vendor validate
+PODMANCMD ?= podman
+IN_CONTAINER = $(patsubst %,%-in-container,$(IN_CONTAINER_TARGETS))
+
+.PHONY: $(IN_CONTAINER_TARGETS)
```

Comment:
I am not sure but should this be:
```suggestion
.PHONY: $(IN_CONTAINER)
```
I think the .PHONY name should match the target name?

Reply:
Yes, you're right. I missed that when I did a minor cleanup and rebase.

Reply:
Should be updated now. Sorry for not applying the suggestion directly, was signed in as a different account.

Reply:
no poblem